gui/src/components/settings/pages/AdvancedSettings.tsx

```diff
@@ -0,0 +1,187 @@
+import { useLocalization } from '@fluent/react';
+import { useState } from 'react';
+import { Typography } from '@/components/commons/Typography';
+import {
+  SettingsPageLayout,
+  SettingsPagePaneLayout,
+} from '@/components/settings/SettingsPageLayout';
+import { WrenchIcon } from '@/components/commons/icon/WrenchIcons';
+import { Button } from '@/components/commons/Button';
+import { SettingsResetModal } from '@/components/settings/SettingsResetModal';
+
+import { invoke } from '@tauri-apps/api/core';
+import { open } from '@tauri-apps/plugin-shell';
+import { error } from '@/utils/logging';
+import { defaultConfig as defaultGUIConfig, useConfig } from '@/hooks/config';
+import { defaultValues as defaultDevConfig } from '@/components/widgets/DeveloperModeWidget';
+import { RpcMessage, SettingsResetRequestT } from 'solarxr-protocol';
+import { useWebsocketAPI } from '@/hooks/websocket-api';
+
+function guiDefaults() {
+  // Destructure the properties to exclude "lang"
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { lang, ...guiDefaults } = defaultGUIConfig;
+
+  // Include "devSettings" which has all the properties of "defaultDevConfig"
+  // @ts-expect-error "devSettings" is not in the "guiDefaults" object but we want to include it (from "defaultDevConfig")
+  guiDefaults.devSettings = defaultDevConfig;
+
+  return guiDefaults;
+}
+
+export function AdvancedSettings() {
+  const { l10n } = useLocalization();
+  const { setConfig } = useConfig();
+
+  const [skipWarningGui, setSkipWarningGui] = useState(false);
+  const [skipWarningServer, setSkipWarningServer] = useState(false);
+  const [skipWarningAll, setSkipWarningAll] = useState(false);
+  const { sendRPCPacket } = useWebsocketAPI();
+
+  const openConfigFolder = async () => {
+    try {
+      const configPath = await invoke<string>('get_config_path');
+      await open('file://' + configPath);
+    } catch (err) {
+      error('Failed to open config folder:', err);
+    }
+  };
+
+  return (
+    <SettingsPageLayout>
+      <form className="flex flex-col gap-2 w-full">
+        <SettingsPagePaneLayout icon={<WrenchIcon></WrenchIcon>} id="advanced">
+          <>
+            <Typography variant="main-title">
+              {l10n.getString('settings-utils-advanced')}
+            </Typography>
+
+            <div className="grid gap-2 mobile:gap-6">
+              <div className="sm:grid sm:grid-cols-[1.75fr,_1fr] items-center">
+                <div>
+                  <Typography bold>
+                    {l10n.getString('settings-utils-advanced-reset-gui')}
+                  </Typography>
+                  <div className="flex flex-col pt-1 pb-2">
```

Comment:
the `pt-1` is kinda fine but what are the `pb-2`s for?

Reply:
This was copied from `InterfaceSettings.tsx` since I used that file as a template, but I'll remove those and change `gap-2` to `gap-4`